tests/notification/base.php

```diff
@@ -66,6 +66,8 @@ protected function setUp()
 			$phpbb_root_path,
 			$phpEx
 		);
+		
+		$phpbb_dispatcher = $this->phpbb_dispatcher = new phpbb_mock_event_dispatcher();
```

Comment:
I did it to make it the same way as all other are doing. Should I still not assign it?